testing/log4cxx/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer:
+# adapted from https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=log4cxx&id=904e7c4f47191e9558cfe26c4d4455a95a0e136d
+
+pkgname=log4cxx
+pkgver=0.10.0
+pkgrel=0
+pkgdesc="A C++ port of the Log4j logging framework."
+url="http://logging.apache.org/log4cxx"
+license="Apache"
+options='!check'  # testsuite in 0.10.0 does not work on alpine
+subpackages="$pkgname-dev"
+depends='apr-util libxml2'
```

Comment:
i think libxml2  and apr-util will be found automatically via the `so:` provider which matches a dependency on a `shared object` (a library) which is provided by a package.

Please feel free to mark this as resolved if it os not the case